scripts/ci/update-github

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -ex
+
+TAG_NAME=$(git tag -l --contains HEAD)
+
+rm -rf dcos-nightly
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git config credential.helper 'cache --timeout=300'
+git clone "https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos" dcos-nightly
+cd dcos-nightly
+
+if [[ $TAG_NAME ]]
+then
+  git checkout -- .
+  git remote add upstream "https://$GIT_USER:$GIT_PASSWORD@github.com/dcos/dcos"
+  git checkout master
+  git checkout -b dcos-ui/$TAG_NAME
+
+  cp -fR ../buildinfo.json packages/dcos-ui/buildinfo.json
+
+  git add "packages/dcos-ui/buildinfo.json"
+  git commit -m "chore(dcos-ui): update package"
+  git push --force origin dcos-ui/$TAG_NAME
```

Comment:
I think force pushing on a tag name is harmful if someone else updated the branch in the meantime, but there is an easy fix for that: [force-with-lease](https://developer.atlassian.com/blog/2015/04/force-with-lease/).

We do a new commit, so I don't think we should even force it.

Reply:
I think being forceful is good because it gives more certainty to the script. We always want this branch to reflect what we want from this Jenkins job. And no real damage to be done - can always use reflog manually if something awful happens and we override something we shouldn't have.

Reply:
Maybe I am too cautious here 🤷‍♂️

Reply:
Force with lease also force pushes, it just reduces the risk to screw something up by a lot and it is to my knowledge a best practice (I got it in my force push alias, too 👍)

Reply:
Ok nice, that makes sense. I've switched it to use --force-with-lease